picard/ui/__init__.py

```diff
@@ -113,9 +118,21 @@ def __init__(self, parent=None):
     def keyPressEvent(self, event):
         if event.matches(QtGui.QKeySequence.Close):
             self.close()
+        elif event.matches(QtGui.QKeySequence.HelpContents) and self.help_url:
+            self.show_help()
         else:
             super().keyPressEvent(event)
 
+    def show_help(self):
+        if self.help_url:
+            url = self.help_url
+            if url.startswith('/'):
+                url = DOCS_BASE_URL + url
+            if url.startswith('goto://'):
```

Comment:
In fact, we could prolly simplify things here, as none of the keys from `PICARD_URLS` contains a `/`, they will never conflict with URLs, what about just checking if `url in PICARD_URLS` in `webbrowser2.open()` and totally drop `goto` method?